Prediction.cs

```diff
@@ -432,7 +235,7 @@ internal static PredictionOutput GetPrediction(PredictionInput input, bool ft, b
             if (ft)
             {
                 //Increase the delay due to the latency and server tick:
-                input.Delay += Game.Ping / 2000f + 0.06f;
+                input.Delay += Game.Ping / 2000f + 0.05f;
```

Comment:
It's actually the correct value. If you calculate it why would it be broscience?